low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -0,0 +1,168 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import os
+import sys
+import errno
+import shutil
+import distutils.dir_util
+from cortx.sspl.bin.sspl_constants import (REPLACEMENT_NODE_ENV_VAR_FILE, PRODUCT_NAME, SSPL_BASE_DIR,
+    file_store_config_path, PRODUCT_BASE_DIR)
+from cortx.sspl.lowlevel.framework.utils.utility import Utility
+from cortx.utils.process import SimpleProcess
+
+
+class PostInstallError(Exception):
+    """ Generic Exception with error code and output """
+
+    def __init__(self, rc, message, *args):
+        self._rc = rc
+        self._desc = message % (args)
+
+    def __str__(self):
+        if self._rc == 0: return self._desc
+        return "error(%d): %s" %(self._rc, self._desc)
+
+
+class SSPLPostInstall:
+    def __init__(self, args: list):
+        """Ssplpostinstall init."""
+        self.args = args
+        self.name = "sspl_post_install"
+        self._script_dir = os.path.dirname(os.path.abspath(__file__))
+        self.RSYSLOG_CONF="/etc/rsyslog.d/0-iemfwd.conf"
+        self.RSYSLOG_SSPL_CONF="/etc/rsyslog.d/1-ssplfwd.conf"
+        self.PACEMAKER_INSTALLATION_PATH="/lib/ocf/resource.d/seagate/"
+
+    def process(self):
+        if not self.args or len({'-p', '-e', '-c'} & set(self.args)) == 0:
+            raise PostInstallError(errno.EINVAL, f"Invalid Argument for {self.name}")
+
+        PRODUCT = ""
+        ENVIRONMENT = ""
+        RMQ_CLUSTER = "true"
+        i = 0
+        while i < len(self.args):
```

Comment:
Script not supposed to take arguments beyond confstor url.  We are not supposed to check in without Conf Stor integration as per Ujjwal's comments in morning. So plan to include ConfStor changes with @mariyappanp help here and then proceed.

Reply:
Change in plan. @mariyappanp will base his patch for confstor on this PR and address these open items. This PR wont be merged as it is currently, given the concerns of incompleteness wrt confstor integration raised by Ujjwal.

Reply:
Done.